src/modules/displayMovieCards.js

```diff
@@ -0,0 +1,37 @@
+import { movieContainer } from "./variables";
+import { createElement } from "./DomUtilities";
+
+const displayMovies = (movies) => {
+    
+  movieContainer.innerHTML = null;
+  movies.forEach((movie, index) => {
+    let movieCard = createElement("div");
+    let movieCardContent = `
+            <img
+              src="https://static.tvmaze.com/uploads/images/medium_portrait/136/342487.jpg"
```

Comment:
## Dynamic content 
Instead of displaying the same content for all of the cards, I suggest that you could update the content dynamically with movies images and title

Thank you!

Reply:
Great! I am about to display it dynamically. Thank you for the notice.